extensions/transferto/action.go

```diff
@@ -46,7 +46,7 @@ func NewTransferAirtimeAction(uuid flows.ActionUUID, amounts map[string]decimal.
 }
 
 // Validate validates our action is valid and has all the assets it needs
-func (a *TransferAirtimeAction) Validate(assets flows.SessionAssets) error {
+func (a *TransferAirtimeAction) Validate(assets flows.SessionAssets, context *flows.ValidationContext) error {
```

Comment:
need some state to prevent loops - tried putting this in `FlowAssets` but turned out to be simpler to avoid race conditions if it's passed around as an arg